testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/AbstractServletsAdapterTest.java

```diff
@@ -65,15 +73,25 @@ protected static WebArchive servletDeploymentMultiTenant(String name, Class... s
     }
 
     protected static WebArchive servletDeployment(String name, Class... servletClasses) {
-        return servletDeployment(name, "keycloak.json", servletClasses);
+        return servletDeployment(name, DEFAULT_ADAPTER_CONFIG, servletClasses);
     }
 
-    protected static WebArchive servletDeployment(String name, String adapterConfig, Class... servletClasses) {
+    protected static WebArchive servletDeployment(String name, Consumer<AdapterConfig> adapterConfig, Class... servletClasses) {
```

Comment:
It seems that `adapterConfig` here is always `null`. Where is this method used from? Why is the `Consumer` parameter introduced?

Reply:
In the first commit of this PR, this is used in _DemoServletAdapterTest_, but the solution for instability of the test is out of scope of this PR. So, it's like a preparation for another related tasks and configure adapter directly from a test. Please, see [here](https://github.com/keycloak/keycloak/pull/8165/commits/d1b77b938f54ef1d1070889eb020a79fdf21b503#diff-a7d3a845d41009122b21855f7fa779bbf324c4e6ba58f4404bf4f58addc89cadR215). @hmlnarik WDYT?

Reply:
The latter would be better, thank you

Reply:
Ok